lib/components/os/resinos.js

```diff
@@ -20,14 +20,10 @@ const Bluebird = require('bluebird')
 const imagefs = require('resin-image-fs')
 
 const utils = require('../../utils')
-
-exports.sshHostOS = async (command, uuid, privateKeyPath) => {
-  const options = [ '-p 22222', `root@${uuid}` ]
-  return utils.ssh(command, privateKeyPath, options)
-}
+const Resinio = utils.requireComponent('resinio', 'sdk')
```

Comment:
Since we're requiring a specific implementation of the resinio component, then maybe worth using `require()` rather than our dynamic wrapper?